src/actions/activity.js

```diff
@@ -3,31 +3,31 @@
 import { ACTIVITY_MESSAGES_LIMIT } from '../global/Constants';
 import { type ActivityLogMessage } from '../types/ActivityLogMessage';
 
-type MessageAddedAction = { +type: 'MESSAGE_ADDED', +message: ActivityLogMessage };
+type MessageAddedAction = { +type: 'MESSAGE_ADDED', +messages: Array<ActivityLogMessage> };
 type StartFetchMessagesAction = { +type: 'START_FETCH_MESSAGES', +limit: number };
 type DoneFetchMessagesAction = { +type: 'DONE_FETCH_MESSAGES', +messages: Array<ActivityLogMessage> };
-type AddDummyMessageAction = { +type: 'ADD_DUMMY_MESSAGE' };
+type AddNewMessageAction = { +type: 'ADD_NEW_MESSAGE', +message: string, +params: any, +interpret: boolean };
 
 export type Action =
   | MessageAddedAction
   | StartFetchMessagesAction
   | DoneFetchMessagesAction
-  | AddDummyMessageAction;
+  | AddNewMessageAction;
 
 export const MESSAGE_ADDED = 'MESSAGE_ADDED';
 export const START_FETCH_MESSAGES = 'START_FETCH_MESSAGES';
 export const DONE_FETCH_MESSAGES = 'DONE_FETCH_MESSAGES';
-export const ADD_DUMMY_MESSAGE = 'ADD_DUMMY_MESSAGE';
+export const ADD_NEW_MESSAGE = 'ADD_NEW_MESSAGE';
 
 /**
  * @desc Action creator for an action that should be called once new activity log message added.
- * @param {ActivityLogMessage} message Activity log message that is added.
+ * @param {ActivityLogMessage} messages Activity log message that is added.
  * @returns {MessageAddedAction} An action.
  */
-export function messageAdded(message: ActivityLogMessage): MessageAddedAction {
+export function messageAdded(messages: Array<ActivityLogMessage>): MessageAddedAction {
```

Comment:
As far as I see from implementation, this method is called with collection of all messages not only those that are added. Consider renaming it to `messagesUpdated` then and change related reducer action handling.

Reply:
I have tested the logic on the app and it was working just fine. Can you test it on your end?